js/ui/windowManager.js

```diff
@@ -2311,8 +2311,6 @@ const CodingManager = new Lang.Class({
         this._rotateInActors.push(dst);
         this._rotateOutActors.push(src);
 
-        dst.rotation_angle_y = direction == Gtk.DirectionType.RIGHT ? -180 : 180;
-        src.rotation_angle_y = 0;
         dst.pivot_point = new Clutter.Point({ x: 0.5, y: 0.5 });
         src.pivot_point = new Clutter.Point({ x: 0.5, y: 0.5 });
```

Comment:
You might also want to set pivot points at the same time `rotation_angle_y` is set too